CanonicalLtd/maas-docs
# Update run script to 2.7.1

Update to the latest run script, v2.7.1.

## QA
Check `./run` command correctly serves the site, and other common ./run commands
